openedx/core/djangoapps/schedules/content_highlights.py

```diff
@@ -62,6 +64,24 @@ def get_week_highlights(user, course_key, week_num):
     return highlights
 
 
+def get_next_section_highlights(user, course_key, target_date):
+    """
+    Get highlights (list of unicode strings) for a week, based upon the current date.
+
+    Raises:
+        CourseUpdateDoeNotExist: if highlights do not exist for the requested date
```

Comment:
This feels a little weird (as opposed to returning None or an empty string to indicate there are no highlights), but I'm not sure if this is just a pattern that is expected.

Reply:
Ahh yeah, it's consistent with the flow control that the other job had. From what I understood, most of the reason is that multiple methods that get called in here can throw this exception so it'll exit early if any of them would return "None" as you mentioned. Saves a couple of if checks.